### DISCUSSION TITLE
How to remove padding-top for q-page-container? / Or how to use space under transparent headerbar?

### DISCUSSION THREAD

**Discussion by HoboDev** [quasarframework/quasar#15762] [General - Components / Directives / etc]
Hi,

I have a project where the headerbar will be transparent or use a backdrop-filter.
Currently I'm stuck on how to use the area below the bar which is usually hidden.

So far I've tried to change the class and putting the header in the q-page-container but that does not lead to any changes.
I'd like to get a fullpage picture above the fold that should start at the top
and have the bar z-indexed above it.

Any pointers are appreciated.

**Comment by cdar** [comment]
I don't know why this is not available to end user in a normal way :/
You may use this "hack":

```
const layoutKey = '_q_l_';
const $layout = inject(layoutKey);
console.log("$layout", $layout);
$layout.header.space = false;
```

**Reply by SaadTariq-FabTechSol** [reply]
How and where??
